CONTRIBUTING.md

```diff
@@ -59,7 +59,9 @@ All dependancies installed? great, now building PouchDB itself is a breeze:
 
 You will now have various distributions of PouchDB in your `dist` folder, congratulations.
 
- * If you are on windows, you will need `node-gyp` to install levelup, visit https://github.com/TooTallNate/node-gyp#installation for installation instructions.
+Note that the source code is in `src/`, which is built by [Rollup](http://rollupjs.org/) as a
+Node module to `lib/`, which is then built by [Browserify](http://browserify.com/) as a browser-ready
+UMD module to `dist/`. All of this logic is in `bin/build.sh`.
```

Comment:
^ Explanation of build setup.

Also I understand this Github diff is impossible to read, but you can just ignore everything in `src`/`lib`, because all the action is in `bin` anyway. `src`/`lib` stuff is just a conversion from ES5 to ES6 (which I did with simple search-and-replace plus some manual tweaks).